### PR TITLE
Added Pull Request Template in markdown

### PR DESCRIPTION
**Pull request by izamglima**
Closes: #159

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+What does this implement/fix? Explain your changes.
+---------------------------------------------------
+…
+
+Does this close any currently open issues?
+------------------------------------------
+…
+
+
+Any relevant logs, error output, etc?
```
Comment (btzr-io) [inline PULL_REQUEST_TEMPLATE.md]: We don't need this section ^^

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+-------------------------------------
+(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)
+
+Any other comments?
```
Comment (btzr-io) [inline PULL_REQUEST_TEMPLATE.md]: We don't need this section ^^

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+---------------------------
+**Operating System:** …
+
+**Platform:** …
+
+**Target Platform:** …
+
+**Toolchain Version:** …
+
+**SDK Version:** …
```
Comment (btzr-io) [inline PULL_REQUEST_TEMPLATE.md]: We don't need a toolchain version or a sdk version neither target plataform.
We should probably just keep the operating system to make things simple.

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (izamglima) [reply PULL_REQUEST_TEMPLATE.md]: okay :)

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (izamglima) [reply PULL_REQUEST_TEMPLATE.md]: okay :)

File: PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (izamglima) [reply PULL_REQUEST_TEMPLATE.md]: okay, done :)

Comment (btzr-io) [reply PULL_REQUEST_TEMPLATE.md]: Awesome thanks!

**Pull request merged**